tools/wptrunner/wptrunner/config.json

```diff
@@ -1,4 +1,4 @@
-{"host": "%(host)s",
+{"host": "web-platform.test",
```

Comment:
Given the goal was to avoid hardcoding the domain everywhere; pretty sure the right fix for this is to make sure we always have an appropriate host provided when we evaluate it (or just drop it if we don't).